Storage.md

```diff
@@ -0,0 +1,307 @@
+Each of the item's elements are added to the container as its children.
+It can be used to display a number of elements grouped together.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    doMakeContainer: true,
+    values: {
+        color: {
+            valueDefault: "red",
+            hasElement: true,
+            element: {}
+        }
+    }
+});
+
+// color is in the container, but speed is not
+ItemsHolder.addItem("speed", {
+    valueDefault: 10,
+    hasElement: true,
+    element: {}
+});
+```
+
+### Display Changes
+
+If you want certain values to be represented differently on the page, make use of `displayChanges`.
+These changes are hardcoded values that replace specified values when element items are updated.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({
+    displayChanges: {
+        Infinity: "INF";
+    }
+});
+
+ItemsHolder.addItem("limit", {
+    value: "4000",
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("limit", "Infinity");
+```
+
+`hasDisplayChange` checks to see if a value has a recorded change.
+
+```typescript
+ItemsHolder.hasDisplayChange("Infinity");
+```
+
+...and `getdisplayChange` returns the entry to replace the value with.
+
+```typescript
+let newValue: string = ItemsHolder.getDisplayChange("Infinity");
+```
+
+### Container Visibility
+
+`hideContainer` hides the container from view.
+
+```typescript
+ItemsHolder.hideContainer();
+```
+
+...while `displayContainer` makes the container visible.
+
+```typescript
+ItemsHolder.displayContainer();
+```
+
+More can be read about ItemsHoldr on its [Readme](https://github.com/FullScreenShenanigans/ItemsHoldr/blob/master/README.md).
+
+# StateHoldr
+
+StateHoldr is a module for tracking changes of items in ItemsHolder in objects called collections.
+Changes are key-value pairs of attributes for items and values.
+A collection is a group of these changes keyed by the respective item name.
+Collections describe how items have changed.
+
+```typescript
+let garage: { [i:string]: any } = {
+    car: {
+        color: "red"
+    }
+}
+```
+
+Collections allow for various attributes for a single item to be grouped together versus storing individual items.
+
+## Prefix
+
+Like ItemsHoldr, StateHoldr has its own prefix property.
+All collections in StateHoldr are stored in ItemsHoldr prepended with the StateHoldr prefix.
+
+```typescript
+let StateHolder = new StateHoldr({
+    ItemsHolder: new ItemsHoldr(),
+    prefix: "StateHolder"
+});
+```
+
+## Collections
+
+The collections the module has recorded are stored in a list keyed by `stateCollectionKeys` in ItemsHolder (no StateHoldr prefix prepended).
+StateHoldr always has one collection as its current collection.
+
+`setCollection` sets the current collection.
+
+```typescript
+StateHolder.setCollection("newCollection", {
+    car: {
+        color: "red"
+    }
+});
+```
```

Comment:
Can you mention that you setting newCollection later will go back to the car/color/red one?